source/overloading.tex

```diff
@@ -2284,7 +2284,7 @@
 and whether a function is deleted\iref{dcl.fct.def.delete}, are ignored.
 So, although an implicit
 conversion sequence can be defined for a given argument-parameter
-pair, the conversion from the argument to the parameter might still
+pair, the conversion from the argument to the parameter can still
```

Comment:
This seems risk rather than capability to me.

Reply:
Reverted.